integration_test/sql/subquery.exs

```diff
@@ -18,6 +18,7 @@ defmodule Ecto.Integration.SubQueryTest do
            TestRepo.all(from p in subquery(query), select: p)
   end
 
+  @tag :map_boolean_in_subquery
```

Comment:
OK, I'll review the MySQL adapter and see if that works. I'll withdraw this tag.